File: SoloLearn Codes/Password Validation/password_validation.cpp
#include <iostream>
#include <regex>
#include <string>
using namespace std;

int main() {
    string inp;
    cin >> inp;
    cout << ((regex_match(inp, regex("((?=.*[!@#$%&*])(?=.*[0-9]).{7,})"))) ? "Strong" : "Weak");
    return 0;
}

File: SoloLearn Codes/Password Validation/PasswordValidation.java
import java.util.regex.*;
import java.util.Scanner;

public class PasswordValidation
{
    public static void main(String[] args) {
        System.out.println(Pattern.matches("((?=.*[!@#$%&*])(?=.*[0-9]).{7,})", new Scanner(System.in).nextLine()) ? "Strong" : "Weak");
	}
}